app/authenticators/custom.js

```diff
@@ -0,0 +1,46 @@
+import Ember from 'ember';
+import Base from 'simple-auth/authenticators/base';
+
+export default Base.extend({
+  tokenEndpoint: 'http://localhost:3001/sessions/create',
+  restore: function(data) {
+    return new Ember.RSVP.Promise(function (resolve, reject) {
+      if (!Ember.isEmpty(data.session_name)) {
+        resolve(data);
+      }
+      else {
+        reject();
+      }
+    });
+  },
+
+  authenticate: function(options) { 
+    var self = this;
+    return new Ember.RSVP.Promise(function(resolve, reject) { 
+    Ember.$.ajax({ 
+      url: self.tokenEndpoint, 
+      type: 'POST', 
+      data: JSON.stringify({
+          username: options.identification,
```

Comment:
ahh, i did not. i thought you were talking about the `identification` part, not the `username` part.